lang/java/pom.xml

```diff
@@ -42,8 +42,8 @@
       To build the avro-mapred module against Hadoop 2 specify
       -Dhadoop.version=2 or leave unspecified to build against Hadoop 1
     -->
-    <hadoop1.version>0.20.205.0</hadoop1.version>
```

Comment:
Yes, I should have just change hadoop2.version to 2.6.0, leaving the hadoop1.version as it is (that is, 2.0.1-alpha)
I will make these changes and commit it.